File: ios/ExpoYandexMobileAdsConfig.swift
import ExpoModulesCore

internal struct ExpoYandexMobileAdsConfig: Record {
  @Field
  var userConsent: Bool = false
    
  @Field
  var locationConsent: Bool = false
    
  @Field
  var enableLogging: Bool = false
    
  @Field
  var enableDebugErrorIndicator: Bool = false
}

File: android/src/main/java/expo/modules/yandexmobileads/ExpoYandexMobileAdsConfig.kt
package expo.modules.yandexmobileads

import expo.modules.kotlin.records.Record

class ExpoYandexMobileAdsConfig: Record {
  val userConsent: Boolean = false
  val locationConsent: Boolean = false
  val enableLogging: Boolean = false
  val enableDebugErrorIndicator: Boolean = false
}